templates/typescript_gapic/_util.njk

```diff
@@ -47,7 +49,7 @@ limitations under the License.
 {%- macro printCommentsForMethod(method) -%}
 {%- set lines = method.comments -%}
 {% for line in lines %}
- *{{ line.replaceAll('*/', '* /').replace(r/\s+$/, '') | safe}}
+    *{{ line.replaceAll('*/', '* /').replace(r/\s+$/, '') | safe | replace(regExp, '{@link $2|$1}')}}
```

Comment:
I'm guessing the extra two spaces before `*` come from here?

Reply:
remove the space could fix